cmd/ls.go

```diff
@@ -129,3 +145,74 @@ func init() {
 
 	lsCmd.Flags().BoolP("long", "l", false, "Long listing")
 }
+
+func listOfEntryNames(entries []*files.Metadata) []string {
+	listOfEntryNames := []string{}
+
+	for _, entry := range entries {
+		switch entry.Tag {
+		case "folder":
+			listOfEntryNames = append(listOfEntryNames, entry.Folder.Name+"    ")
```

Comment:
Correct. That avoids having elements in side-by-side columns from being right next to each other.